DeMaTO/.htaccess

```diff
@@ -0,0 +1,53 @@
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .jsonld
+# Rewrite engine setup
+RewriteEngine On
+
+# Forward to most recent version
+
+RewriteRule ^$ v1.0 [L]
+
+# Version-specific redirects
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^([^/]+)$https://github.com/sogolsmansouri/sogolsmansouri.github.io/blob/main/index.html [R=303,L]
```

Comment:
This will not redirect to what you are aiming for. 
You should publish this page through github pages and then add there the redirection link

Reply:
Can you review #3409 if that's okay we can move on with that. Thank you!

Reply:
ok, I have. I will close this one